### ISSUE TITLE
JetWooBuilder Added to Cart Notice Widget

### ISSUE THREAD

**Issue by thorsolutions** [CrocoBlock/suggestions#2249]
Currently by the looks of it, the added to cart section shows up on the top of the Single Product Page, there doesn't seem a way to edit the location of this using any widget or settings.

![image](https://user-images.githubusercontent.com/16325700/101008030-9add8d00-3587-11eb-9bd2-19cca65ccbb6.png)

Ideally, there should be a widget which allows us to choose where we want this notice to show and it should override the default location and let us add it in custom locations on the single product page.

**Comment by iamdanbarrett** [comment]
I agree! This is a must-have to complete the WooBuilder set. Currently, there is no control over this but with CSS - it would be good to be able to switch off the default one (or it does this automatically when you add the notification widget) and appear only when a product is added to the cart.

**Comment by 79ho3ein** [comment]
It would be great if it was displayed as a pop-up.

**Comment by lukepatrickillustrations** [comment]
This is such an important feature. I would be absolutely amazing if there was a widget for this. A popup would be the best but I would settle for just a widget for the time being.

**Comment by 79ho3ein** [comment]
+1

**Comment by 79ho3ein** [comment]
Does jet woobuilder have a plan to add this feature? Many users are waiting for this feature. Please prioritize it. This feature is extremely necessary.
Thanks to the crocoblock team.

**Comment by UraraReika** [comment]
@79ho3ein , yes we have.

**Comment by startupweb** [comment]
So excited for this!  Definitely an important feature for all my e-commerce websites. 

Client's usually miss the notification when AJAX add to cart is on.

When AJAX is off, then the notification appears in a weird position breaking the Elementor design. 

Thank you Crocoblock.

**Comment by 79ho3ein** [comment]
Yes, exactly, I hope crocoblock adds this feature soon. I'm counting the time to add it.
@UraraReika I'm sorry ... is it clear when this feature will be released? Many users are waiting for this important feature.

Thank you for your efforts.

**Comment by 79ho3ein** [comment]
@UraraReika 
excuse me ...
I had two question about this feature (WooCommerce notifications). Thank you for answering.
1- Does this feature display WooCommerce notifications as a pop-up?
2- Does this feature only display the notification of a product being added to the cart, or all WooCommerce notifications?
For example, it also displays the "Discount Coupon is not valid" notice on the payment slip?
Or, for example, notifications related to payment fields ؟

**Comment by 79ho3ein** [comment]
hello, is there any news on this?

**Comment by shopakr** [comment]
WooCommerce Pop Up Notification option would be amazing.  Hoping to see ability to control which notifications pop-up, and a timed exit of pop-up feature. I found a third party plugin that does all of this (https://wpfactory.com/item/popup-notices-for-woocommerce/) but who wants to manage another plugin.

**Comment by 79ho3ein** [comment]
hi @shopakr 
Yes, there is currently only one plugin for displaying WooCommerce notifications as a pop-up, which is highly inefficient. Unfortunately, I am currently using this plugin and for this reason I am facing many problems.
I just hope this feature is added to jet woobuilder soon to solve a big problem. This is really needed

**Comment by l3ynad** [comment]
almost one full year on this and no update

**Comment by webcoreinc** [comment]
Croco woo builder is missing ALOT of features as well as full of bugs.

What you need to do is this:

1. Hide message wrapper w css
2. Add a plugin for a sidecart or popup. It will require another plugin.

**Comment by Robinsgood** [comment]
+1

**Comment by borjamerino-v** [comment]
+1

**Comment by Joliamendi** [comment]
Hi @UraraReika, @Floydtm

Do you have any plans to add woo notice popup to jet woobuilder? It has been almost 4 years since this feature was requested. Is there any good news?
